assets/src/apps/authoring/components/SequenceEditor/SequenceEditor.tsx

```diff
@@ -52,12 +52,15 @@ const SequenceEditor: React.FC<any> = (props) => {
   const handleItemAdd = async (
     parentItem: SequenceEntry<SequenceEntryChild> | undefined,
     isLayer = false,
+    isBank = false,
   ) => {
     let layerRef: string | undefined;
     if (parentItem) {
       layerRef = parentItem.custom.sequenceId;
     }
-    const newTitle = `New ${layerRef ? 'Child' : ''}${isLayer ? 'Layer' : 'Screen'}`;
+    const newTitle = `New ${layerRef && !isBank ? 'Child' : ''}${
+      isLayer ? 'Layer' : isBank ? 'Question Bank' : 'Screen'
```

Comment:
not usually a good idea to keep going with nested tertiary conditions. it looks like maybe one is missing from this? anyway if you have more than one, then you should break it out into a variable